<launch>
    <arg name="namespace" default="uwb_simulation_node" />
    <arg name="label_name" default="x500_0" />

    <node pkg="uwb_simulation"
        exec="uwb_simulation_node"
        namespace="$(var namespace)"
        output="screen">
        <param name="label_name" value="$(var label_name)" />
    </node>
</launch>